### PR TITLE
aries-vcx: Fix proof presentation where aries-vcx is verifier and acapy is prover

### PR DESCRIPTION
**Pull request by mirgee**
Signed-off-by: Miroslav Kovar <miroslavkovar@protonmail.com>

### CODE REVIEW

File: aries-backchannels/aries-vcx/Cargo.toml [hunk 1]
```diff
@@ -22,4 +22,4 @@ pickledb = "0.4.1"
 uuid = { version = "0.8.2", features = ["serde", "v4"] }
 futures-util = "0.3.15"
 futures = "0.3.15"
-aries-vcx = { git = "https://github.com/hyperledger/aries-vcx", rev = "a4fe9fb" }
+aries-vcx = { git = "https://github.com/hyperledger/aries-vcx", rev = "0e47a081f" }
```
Comment (ianco) [inline aries-backchannels/aries-vcx/Cargo.toml]: This revision isn't working (commit not in aries-vcx repo)
I'm testing with `edc78bd15355ce1efc9dd338fd3a6cd4f93ad287`

**Review by ianco — CHANGES_REQUESTED** [review]
ok I'm not able to get the aries-vcx backchannel to build :-(

File: aries-backchannels/aries-vcx/Cargo.toml (hunk 1 above)
Comment (mirgee) [reply aries-backchannels/aries-vcx/Cargo.toml]: Updated the aries-vcx revision, make sure you have commit `e86e46d62adb41b29facce5d9bf25894da63f7c3` of this branch (`mirgee:feature/aries-vcx-backchannel`) checked out.

**Comment by swcurran** [comment]
Running off this branch from before the two most recent commits and I'm seeing things looking way better.  

Current main:

```
2 features passed, 2 failed, 4 skipped
9 scenarios passed, 32 failed, 69 skipped
198 steps passed, 32 failed, 765 skipped, 0 undefined
Took 34m48.777s
```

With this branch (pre-commits):

```
2 features passed, 2 failed, 4 skipped
18 scenarios passed, 23 failed, 69 skipped
257 steps passed, 23 failed, 715 skipped, 0 undefined
Took 15m30.969s
```

Will try again with the latest commits...

Please let me know when you are happy with this PR, @mirgee .

**Review by ianco — APPROVED** [review]
3 features passed, 0 failed, 5 skipped
20 scenarios passed, 0 failed, 90 skipped
165 steps passed, 0 failed, 830 skipped, 0 undefined

**Pull request merged**